### PR TITLE
Redux task

### PR DESCRIPTION
**Pull request by Moraheem98**
- PR for useSelector hook and dispatch implementation for switchBanner Component

### CODE REVIEW

File: task-2/src/store/actionCreaters/actionCreators.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import {
```
Comment (Dmitry-White) [inline task-2/src/store/actionCreaters/actionCreators.js]: There's a typo in the folder name.

File: task-2/src/store/allReducers/rootReducer.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import {
```
Comment (Dmitry-White) [inline task-2/src/store/allReducers/rootReducer.js]: Consider renaming the containing folder to "reducers".

File: task-2/src/components/Home/index.js [hunk 3]
```diff
@@ -11,10 +13,14 @@ import './index.css';
 
 export const Home = () => {
 	const [openModal, setOpenModal] = useState(false);
-	const [selectedMovie, setSelectedMovie] = useState(undefined);
+
 	const [displayButton, setDisplayButton] = useState(false);
 	const show = () => setDisplayButton(true);
 
+	const selectedMovie = useSelector(
+		(state) => state.switchBannerReducer.selectedMovie,
```
Comment (Dmitry-White) [inline task-2/src/components/Home/index.js]: Consider creating a separate folder "selectors" under "src/store".
Then name and move all the arrow functions you pass to `useSelector` hooks there.
Think of them. the same way as you think of Action Creators:
Action Creators = functions that return Actions
Selectors = functions that (select and) return Parts of State

File: task-2/package.json [hunk 4]
```diff
@@ -16,10 +16,15 @@
...
 	"dependencies": {
+		"@reduxjs/toolkit": "^1.7.1",
 		"prop-types": "^15.8.1",
 		"react": "^17.0.2",
 		"react-datepicker": "^4.6.0",
-		"react-dom": "^17.0.2"
+		"react-dom": "^17.0.2",
+		"react-redux": "^7.2.6",
+		"redux": "^4.1.2",
+		"redux-devtools-extension": "^2.13.9",
```
Comment (Dmitry-White) [inline task-2/package.json]: This package is not needed and is deprecated.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]
Looking forward to the rest of the updates!

File: task-2/src/store/actionCreaters/actionCreators.js (hunk 1 above)
Comment (Moraheem98) [reply task-2/src/store/actionCreaters/actionCreators.js]: Typo fixed!

File: task-2/src/store/allReducers/rootReducer.js (hunk 2 above)
Comment (Moraheem98) [reply task-2/src/store/allReducers/rootReducer.js]: File name changed!

File: task-2/package.json (hunk 4 above)
Comment (Moraheem98) [reply task-2/package.json]: Removed!

File: task-2/src/components/Home/index.js (hunk 3 above)
Comment (Moraheem98) [reply task-2/src/components/Home/index.js]: - Selector file created.
- Selector has been separated. 
- Home component and movieDetailsBanner has been updated with the appropriate selectors.

**Review by Dmitry-White — APPROVED** [review]

**Pull request merged**